tests/multi-select.spec.ts

```diff
@@ -0,0 +1,254 @@
+})
+
+test('Given a multi-select with button, there is a button at the bottom', async ({page}) => {
+    const multiselect = page.getByTestId('multiselect-with-button')
+    await expect(multiselect).toBeVisible()
+    await multiselect.click()
+    await expect(multiselect.locator('v-select-button').first()).toBeVisible()
+    await expect(multiselect.locator('v-select-button').first()).toHaveText(/TAKE PHOTO/)
+    await expect(multiselect.locator('v-select-button').first().locator('v-icon').first()).toBeVisible()
+    
+    const buttonClickevent = waitForCustomEvent(page, 'button-click')
+    await multiselect.locator('v-select-button').first().click()
+    expect(buttonClickevent).toBeTruthy()
+
+})
+
+test('Given a multi-select with a header, there is a header in the optiosn list', async ({page}) => {
+    const multiselect = page.getByTestId('multiselect-header')
+    await expect(multiselect).toBeVisible()
+    await multiselect.click()
+    const optionsContainer = multiselect.locator('.options-container').first()
+    await expect(optionsContainer.locator('span', {hasText: 'Emotions'})).toBeVisible()
+})
+
+test('Given a disabled multi-select, the element is not editable', async ({page}) => {
+    const multiselect = page.getByTestId('multiselect-disabled')
+    await expect(multiselect).toBeVisible()
+
+    await multiselect.click()
+    expect(multiselect.locator('.options-container').first()).not.toBeVisible()
+    await expect(multiselect.locator('input').first()).toBeDisabled()
+})
+
+test('Given a readonly multi-select, the element is not editable', async ({page}) => {
+    const multiselect = page.getByTestId('multiselect-readonly')
+    await expect(multiselect).toBeVisible()
+    await multiselect.click()
+    await expect(multiselect.locator('.options-container').first()).not.toBeVisible()
+    // only exists if readonly is true
+    await expect(multiselect.locator('input').first()).toHaveAttribute('readonly', '')
+})
+
+test('Given a clearable=false state, the multi-select does not have the clear all option', async ({page}) => {
+    const multiselect = page.getByTestId('multiselect-not-clearable')
+    await expect(multiselect).toBeVisible()
+    await multiselect.click()
+    await expect(multiselect.locator('.options-container').first()).toBeVisible()
+    await expect(multiselect.locator('.options-container').first().locator('button', {hasText: 'Clear all'})).not.toBeVisible()
+})
+
+test('Given a tooltip and state=info, the select should show an info icon with a tooltip', async ({page}) => {
+    const multiselect = page.getByTestId('multiselect-info-tooltip')
+    await expect(multiselect).toBeVisible()
+    await expect(multiselect.locator('v-tooltip').first()).toBeVisible()
+    await expect(multiselect.locator('v-tooltip').first().locator('div').first()).toHaveClass(/icon-info-outline/)
+    await multiselect.locator('v-tooltip').first().hover()
```

Comment:
don't think you need to test the tooltip's functionality (should be covered in tooltip tests) but doesn't hurt much either